### ISSUE TITLE
Cosmos GraphDb Emulator Support

### ISSUE THREAD

**Issue by SebastianAtWork** [microsoft/vscode-cosmosdb#1248]
Hi,
The local cosmos emulator supports the graph / gremlin interface and it would be really cool if this extension could support that too. It would be (to my knowledge) the only way to show the emulator graph locally. I am already trying to localte the neccessary changes but since im new to extensions, there are probably people who are better suited. From what i´ve seen so far, the problem is that it assumes ssl for connecting to gremlin, yet the local emulator does not support that. It has to be connected to by the https protocol but with useSSL: false. At least when connecting from C#.

**Comment by StephenWeatherford** [comment]
The last time we looked into this, the emulator explicitly said that it does not support graphs or tables.  Is that no longer the case?

**Comment by papa-pep** [comment]
The emulator now supports graphs. There is an option to connect to a graph, but I'm unable to query the emulator graph using that option. "unexpected server response (200)"

**Comment by lurock** [comment]
Yes this would be great if we could disable the option that requires SSL in the extension settings so we can connect to the Cosmos DB emulator that supports running the graph/gremlin database on a non-secure port.

**Comment by hummli** [comment]
Is there any update on this?

**Comment by hummli** [comment]
Is there any update on this?

**Comment by SebastianAtWork** [comment]
Now a year later ... still relevant XD Any updates?
